app/javascript/submission_handling/polling.js

```diff
@@ -2,6 +2,8 @@
 import 'jquery'
 import 'bootstrap'
 
+const defaultIndex = 0;
```

Comment:
would this "back_forward" value ever be at `1`? or some other key? If not I'd suggest something like "navigationKeyLocation" or whatever?

Reply:
I don't know. Maybe just `expectedArrayPosition`? idk

Reply:
I think it would be 0 in our case, `performance.getEntriesByType("navigation")` returns a list of PerformanceEntry objects for every mark set. https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntriesByType. But when no marks set, it seems to contain a single object that has the navigation information for the window. 

<img width="786" alt="Screen Shot 2020-11-03 at 12 26 53 PM" src="https://user-images.githubusercontent.com/6477867/98019661-063d0f00-1dd0-11eb-8a73-05308a29eba3.png">

Reply:
right - so that's what you expect the position of the value to be in the array right? Just a suggestion about a name change as `defaultIndex` to me isn't super clear

Reply:
oh yeah I could have change that, sorry I forgot about this one, will make that change in a another PR

Reply:
it's fine I think